### DISCUSSION TITLE
Your Rest API Source doesn't work

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#833] [General]
It's not possible to import really large datasets via JSON files, so we need to import with a Rest API. We followed the directions here: https://docs.orama.com/cloud/data-sources/custom-integrations/rest-apis - After many failed attempts, we just copied and pasted the code from the instructions page (using our index and API key), which just has 2 objects in the array, and even that didn't work. Nothing gets populated. Interestingly, when we tested a JSON file again, it worked fine, but when we attempted to import the same exact data with the Rest API, it failed again.

**Comment by micheleriva** [comment]
Hi @osseonews, would you join our community Slack at https://orama.to/slack? We'll be happy to give support here!

**Comment by osseonews** [comment]
OK, sure will do. BTW, we got it work by just using the standard fetch call to the endpoint, without the SDK (I.e. CloudManager } from "@oramacloud/client). So maybe there is just some bug in the SDK. I don't know.
